Add JSON Slack rule example here from Slack docs (#139)
JupiterOne/docs: docs/schemas/alert-rule.md

> Sends a Slack message to a given Slack webhook URL.

| Property                | Type     | Description                                                                           |
| ----------------------- | -------- | --------------------------------------------------------------------------------------|
| `integrationInstanceId` | `string` | The `id` of the JupiterOne Jira integration that should be used to create the ticket. |
| `type`                  | `string` | Rule operation action type: `SEND_SLACK_MESSAGE`                                      |
| `channels`              | `string` | A string or list of strings begining with a `#` denoting Slack channels to send to    |
| `webhookUrl`            | `string` | Webhook URL for the account/channel that this message should be delivered to.         |

Example:

> Once the integration has been configured, copy the integration ID from the
> integration instance page. For example, if the integration instance URL looks
> like this:
> 
> `https://apps.us.jupiterone.io/integrations/slack/configuration/d1549f40-b9fd-447a-bec5-4360c9ca7e8c`
> 
> Then the integration instance ID is `d1549f40-b9fd-447a-bec5-4360c9ca7e8c`.

Reviewer: The UI has been improved so that users don't have to copy from the URL.

![Screen Shot 2020-12-03 at 5 48 37 PM](https://user-images.githubusercontent.com/2571/101097895-0166a900-3590-11eb-9d8b-040cbeeebfab.png)

```suggestion
Once the integration has been configured, copy the integration ID from the
integration instance page. It will look something like `d1549f40-b9fd-447a-bec5-4360c9ca7e8c`.
```

> 1. Configure a rule with the `SEND_SLACK_MESSAGE` action and specify the
>    `integrationInstanceId` and `channels` properties with the value being the ID
>    specified in the URL above. Example alert rule configuration with the

Reviewer: ```suggestion
1. Configure a rule with the `SEND_SLACK_MESSAGE` action and specify the
   `integrationInstanceId` with the unique identifier of the integration and `channels`
   denoting the destinations. Example alert rule configuration with the
```
